tests/testsuite.src/syn_occurs.at

```diff
@@ -501,3 +501,107 @@ AT_CHECK([$COMPILE_ONLY -std=mf -fodoslide prog.cob ], [0], [], [])
 
 AT_CLEANUP
 
+
+AT_SETUP([SET to out-of-range literal])
+AT_KEYWORDS([SET])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       DATA             DIVISION.
+       WORKING-STORAGE  SECTION.
+       01 G.
+         02 X           PIC X OCCURS 10 INDEXED I.
+       PROCEDURE        DIVISION.
+           SET I TO ZERO.
+           SET I TO 11.
+           STOP RUN.
```

Comment:
please add a `SET I TO lvl-78-containing-12` and possibly a `01-constant-as-13` to have those paths tested as well